templates/security/cve/cve.html

```diff
@@ -333,26 +339,33 @@ <h2>References</h2>
         <ul>
           {% if cve.references %}
             {% for reference in cve.references %}
-              
-                <li><a href="{{ reference }}">{{ reference }}</a></li>
-    
+
+              <li>
+                <a href="{{ reference }}">{{ reference }}</a>
+              </li>
+
             {% endfor %}
           {% else %}
             <li>
-              <a href="https://cve.mitre.org/cgi-bin/cvename.cgi?name={{ cve.id }}">
-                MITRE
-              </a>
+              <a href="https://cve.mitre.org/cgi-bin/cvename.cgi?name={{ cve.id }}">MITRE</a>
             </li>
           {% endif %}
+
+          {% if cve.notices_ids %}
+            {% for notice in cve.notices_ids %}
+              {% if 'LSN' in notice %}
```

Comment:
Why only notices with the lsn prefix?

Reply:
It looks like the only ones not included in `cve.references` are the `LSN` notices. `cve.notices_ids` includes all notices, so this would lead to duplicates